docs/agent/changelog.md

```diff
@@ -6,6 +6,14 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.2.0 - \[2024-08-29\]
+
+- Added `--url` agent CLI flag for simpler creation of HTTP(S), TLS, and TCP protocol endpoints.
+- Added `--metadata` and `--description` agent CLI flags when using `--url`.
+- Deprecated `--domain`, `--scheme`, and `--remote-addr` agent CLI flags.
+- Added `endpoints:` as a new key in the agent config for endpoint creation.
```

Comment:
Should be a subsection underneath a larger "created new configuration version v3"

 - v2 still supported and has tunnels only
 - v3 has tunnels and endpoints, but tunnels are deprecated for config version 3
 - v3 agent configuration (not tunnels/endpoints) located underneath `agent` parent field. Some fields renamed.